deps.edn

```diff
@@ -1,18 +1,27 @@
 {:paths   ["src" "classes"]
 
  :deps
-          {org.clojure/clojure               {:mvn/version "1.10.1"}
-           clj-time                          {:mvn/version "0.14.2"}
-           javax.servlet/servlet-api         {:mvn/version "2.5"}
-           org.apache.activemq/activemq-core {:mvn/version "5.7.0"}
-           org.slf4j/slf4j-simple            {:mvn/version "1.7.30"}
-           amperity/vault-clj                {:mvn/version "0.5.1"}
-           org.apache.commons/commons-email  {:mvn/version "1.5"}
-           com.taoensso/timbre               {:mvn/version "4.10.0"}}
+          {org.clojure/clojure                   {:mvn/version "1.10.1"}
+           org.clojure/data.json                 {:mvn/version "1.0.0"}
+           javax.servlet/servlet-api             {:mvn/version "2.5"}
+           org.apache.activemq/activemq-core     {:mvn/version "5.7.0"}
+           org.apache.tika/tika-core             {:mvn/version "1.24"}
+           org.slf4j/slf4j-simple                {:mvn/version "1.7.30"}
+           amperity/vault-clj                    {:mvn/version "0.5.1"}
+           org.apache.commons/commons-email      {:mvn/version "1.5"}
+           com.taoensso/timbre                   {:mvn/version "4.10.0"}
+           com.google.cloud/google-cloud-storage {:mvn/version "1.108.0"}}
 
- :aliases {:test     {:extra-paths ["test"]
-                      :extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.629"}}
-                      :main-opts   ["-m" "kaocha.runner"]}
-           :uberdeps {:extra-deps {uberdeps {:mvn/version "0.1.10"}}
-                      :main-opts  ["-m" "uberdeps.uberjar"]}}
+ :aliases {:test        {:extra-paths ["test"]
+                         :extra-deps  {lambdaisland/kaocha               {:mvn/version "1.0.629"}}
+                         :main-opts   ["-m" "kaocha.runner" "unit"]}
+           :integration {:extra-paths ["test"]
+                         :extra-deps  {lambdaisland/kaocha               {:mvn/version "1.0.629"}}
+                         :main-opts   ["-m" "kaocha.runner" "integration"]}
```

Comment:
Maybe we can just combine these into one and just pass the "kind" via the command line? (See https://github.com/broadinstitute/wfl/pull/43/files#diff-29bd8491c570ce25397243cf594a6245L77 for an example?)

Reply:
(Just stolen from your PR, thanks for the example 😄 )